pkg/workos_errors/errors_test.go

```diff
@@ -0,0 +1,52 @@
+package workos_errors_test
+
+import (
+	"fmt"
+	"github.com/workos-inc/workos-go/internal/workos"
+	"github.com/workos-inc/workos-go/pkg/workos_errors"
+	"net/http"
+	"testing"
+)
+
+func TestIsBadRequest(t *testing.T) {
+	type args struct {
+		err error
+	}
+	tests := []struct {
+		name string
+		args args
+		want bool
```

Comment:
nit: We use `expected` instead of `want` in our test suite. Could we `s/want/expected` to be consistent?